src/tasks.py

```diff
@@ -286,6 +286,32 @@ def get_depth_by_time_at_point(model_id: int, lat: float, lng: float) -> DepthTi
     return DepthTimePlot(depths, times)
 
 
+@app.task(base=OnFailureStateTask)
+def retrieve_medusa_input_parameters(scenario_id: int) -> Optional[Dict[str, Union[str, float]]]:
+    """
+    Retrieve input parameters for the current scenario id.
+
+    Parameters
+    ----------
+    scenario_id: int
+        The scenario ID of the pollution model run
+
+
+    Returns
+    -------
+    Dict[src, Union[str, float]]
+        A dictionary contain information from Rainfall MEDUSA 2.0 database or None if scenario does not exist.
+    """
+    # Get rainfall information
+    medusa_rainfall_event = retrieve_input_parameters(scenario_id)
+
+    # Return a dictionary format of these parameters or None if no ID found
+    if medusa_rainfall_event is None:
+        return None
+
```

Comment:
I couldn't get this part of the code to run. If I put in an invalid scenario_id like `9999` into `retrieve_input_parameters` then it raises a `TypeError`. Your documentation says it should raise `FileNotFoundError` and this section of code suggests it should return `None`.

I like the idea of it returning `None` but the implementation and documentation of `retrieve_input_parameters` isn't consistent yet